### ISSUE TITLE
Private Transactions

### ISSUE THREAD

**Issue by trymeouteh** [Minds/crypto#1]
From what I know the minds token is based on the ethereum blockchain. And with a block explorer the minds token is traceable meaning if I bought tokens and sent them to Alice since I want to support Alice or if I want to pay for Bobs exclusive content

**Comment by Serkan-devel** [comment]
There already are offchain-transactions, which are more private

**Comment by trymeouteh** [comment]
> There already are offchain-transactions, which are more private

Can I use the Ethereum Minds Token on an Off-Chain transactions? From what I understand On-Chain are the Ethereum Minds Token and Off-Chain is the old system before that is not blockchain or cryptocurrency based.

**Comment by Serkan-devel** [comment]
But Ethereum isn't designed to be anonymous. It's pseudonymous

**Comment by trymeouteh** [comment]
> But Ethereum isn't designed to be anonymous. It's pseudonymous

Isn't this an issue for privacy? I know Ethereum is not mean't to be anonymous but if I wanted to pay to access exclusive content from a channel using my Minds Tokens someone can trace my transaction and know what I pay to see on Minds.

**Comment by trymeouteh** [comment]
Can you convert your On-Chain funds into Off-Chain funds and have the ability to use Off-Chain funds for tips, accessing exclusive content, boosting, etc?

The transparency argument is a good point since others can see were a content creator get their funds from and how they spend their funds on Minds.

**Comment by Serkan-devel** [comment]
I don't think minds.com wants to run Offchain tokens at all, which is why there is no option to convert it back and offchain is limited to 5 tokens when spent

**Comment by trymeouteh** [comment]
> I don't think minds.com wants to run Offchain tokens at all, which is why there is no option to convert it back and offchain is limited to 5 tokens when spent

I agree I think they want to use the blockchain so in the event that minds is ever hacked it would be harder to steal funds.

Is it possible to create multiple wallets in one minds account. One wallet for receiving funds as a content creator and another wallet you can fund privately and spend on exclusive content and sending tips to contents creators you like?

**Comment by Serkan-devel** [comment]
>Is it possible to create multiple wallets in one minds account. One wallet for receiving funds as a content creator and another wallet you can fund privately and spend on exclusive content and sending tips to contents creators you like?

It is possible to switch between onchain wallets on your minds account.
But it's easy to see transactions between two exact wallets and the amount exchanged, be it between the account to receive fund and the one to spend it. All of that is visible and traceable
